src/content/docs/en/reference/routing-reference.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Routing Reference
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 4
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+There is no separate routing configuration in Astro. Every [supported page file](/en/basics/astro-pages/#supported-page-files) located within the special `src/pages/` directory creates a single page route, or multiple page routes dynamically when the file name contains a [parameter](#params).
+
+
+## `getStaticPaths()`
+
+**Type:** `(options: GetStaticPathsOptions) => Promise<GetStaticPathsResult> | GetStaticPathsResult`
+
+If a page uses dynamic params in the filename, that component will need to export a `getStaticPaths()` function.
```

Comment:
I think the right thing would be to spell the full word (parameters) or then reference the property with code syntax, right?
```suggestion
If a page uses dynamic parameters in the filename, that component will need to export a `getStaticPaths()` function.
```

```suggestion
If a page uses dynamic `params` in the filename, that component will need to export a `getStaticPaths()` function.
```